client/html/templates/catalog/detail/body-standard.php

```diff
@@ -236,7 +236,7 @@
 
 
 						<div class="stock-list">
-							<div class="articleitem stock-actual"
+							<div class="articleitem <?= !in_array( $this->detailProductItem->getType(), ['select', 'group'] ) ? 'stock-actual' : '' ?>"
```

Comment:
Hi, sorry.

Just now i miss out this file
client/html/templates/catalog/detail/body-standard.php